Makefile

```diff
@@ -56,3 +56,13 @@ publish:
 clean:
 	sudo $(RM) -r ./artifacts
 	$(RM) $(MAN_PAGES)
+
+alias-latest:
+	ln -s "$(PWD)"/artifacts/bin/test-reporter-$(VERSION)-darwin-amd64 "$(PWD)"/artifacts/bin/test-reporter-latest-darwin-amd64
+	ln -s "$(PWD)"/artifacts/bin/test-reporter-$(VERSION)-linux-amd64 "$(PWD)"/artifacts/bin/test-reporter-latest-linux-amd64
+
+release:
+	$(MAKE) build-all
+	$(MAKE) alias-latest
+	$(MAKE) publish
+	$(MAKE) clean
```

Comment:
This can be 

```makefile
release: build-all alias-latest publish
```

(Usually I wouldn't include a `clean` at the end of other targets.)